app/controllers/api/v01/bibs_controller.rb

```diff
@@ -64,36 +65,47 @@ def create_or_update_teacher_sets
         )
       rescue => exception
         log_error('create_or_update_teacher_sets', exception)
-        AdminMailer.failed_bibs_controller_api_request(@request_body, "One attribute may be too long.  Error: #{exception.message[0..200]}...", action_name, teacher_set).deliver
+        AdminMailer.failed_bibs_controller_api_request(@request_body, 
+                                                       "One attribute may be too long.  Error: #{exception.message[0..200]}...", 
```

Comment:
Let's move these to four spaces right of the start of line -- the extreme indentation is a bit hard to read.

Reply:
Let's use 
`Layout/ArgumentAlignment:
    EnforcedStyle: with_fixed_indentation
`